rest-api-spec/src/main/resources/rest-api-spec/test/indices.get/10_basic.yml

```diff
@@ -52,6 +52,35 @@ setup:
   - is_true: test_index.settings
   - is_true: test_index.mappings
 
+---
+"Test include_type_name":
+  - skip:
+      version: " - 6.99.99"
+      reason: the include_type_name parameter is not backported to pre 7.0 versions yet
+
+  - do:
+      indices.get:
+        include_type_name: true
+        index: test_index
+
+  - is_true: test_index.mappings
+  - is_true: test_index.mappings.type_1
+
+  - do:
+      indices.get:
+        include_type_name: false
+        index: test_index
+
+  - is_true: test_index.mappings
+  - is_false: test_index.mappings.type_1
+
+  - do:
+      indices.get:
+        index: test_index
+
+  - is_true: test_index.mappings
```

Comment:
I guess this case will need to be split out when backporting to 6.x, since the default will be different.

Reply:
probably yes, not sure yet though...